### ISSUE TITLE
(Autotest) control files depending on Avocado libs

### ISSUE THREAD

**Issue by clebergnu** [avocado-framework/avocado-vt#1512]
Hi,

I've come across commits such as 7b7c5960ede3d2201775d1fd8f47c1df8d7d413c, that switches to using Avocado libs on **autotest** control files.

While I do understand, and welcome, and appreciate, the moving forward of the Python 3 port, and consequently the removal of Autotest dependencies, I don't see the point in removing Autotest dependencies from Autotest control files.  The control files should either be removed completely, or be left alone, to be executed by Autotest only, and not by a combination of Autotest + Avocado.

The current state of things cause failures when running any tests that have "autotest_control" as  "type".

**Comment by lmr** [comment]
Yep, agreed. I'd vote for removing the autotest control subtest.
